lib/benchmarker.js

```diff
@@ -37,164 +39,267 @@ module.exports = function ( config ) {
 			'middle'       : '│'
 		},
 
-		'style' : { 'head' : [ 'white' ] }
+		'style' : {
+			'head' : [ 'white' ]
+		}
 
 	} );
 
-	// plugin to get files with specific pattern or extension
-	function parseExec () {
+	this.host = [ config.server.host, config.server.port ].join( ':' );
 
-		var host = [ config.server.host, config.server.port ].join( ':' );
+	// path to the LUA file that is going to be reused on every 'wrk' execution
+	this.reqPath = path.join( __dirname, './request.lua' );
 
-		// path to the LUA file that is going to be reused on every 'wrk' execution
-		var reqPath = path.join( __dirname, './request.lua' );
+	// path for the request LUA template
+	this.reqTpl = path.join( __dirname, './requestTpl.txt' );
```

Comment:
this is the file path for the request template. we use this as a generic format to request.
https://github.com/Darbiol/wrk-node/commit/ad103d4f583ab2fcc02e509568b9337bd4dcf397#diff-7263807df42494852f10f5b0049fa969R127